dev/PushNotifications/PushNotifications.idl

```diff
@@ -76,6 +71,10 @@ namespace Microsoft.Windows.PushNotifications
     [feature(Feature_PushNotifications)]
     runtimeclass PushNotificationManager
     {
+        // Checks to see if the APIs are supported for the Desktop app
```

Comment:
@jonwis @brflynn There are some minor idl changes here. Just wanted to check that we are good here.

Reply:
I don't _mind_ exposing the ctor - it would make testing of app code easier, as the ISV can just create a new Args and hand it to its Args-handling method, rather than having to invent a wrapper layer.

Reply:
Right. I was just going with correctness here since it wasn't really needed from the projection perspective. In the past API review always erred on the side of exposing projections only if absolutely needed. Unless you do feel super strongly here, I will go ahead and remove since the test scenario can still be exercised by the developer using an E2E test.